submissions/xT0PERx/js-dom/code.js

```diff
@@ -0,0 +1,65 @@
+  },
+];
+
+document.querySelector(".menu-list").addEventListener("click", function (e) {
+  if (e.target && e.target.nodeName === "LI") {
+    switch (e.target.className) {
+      case "menu-item-1":
+        showContent("menu-item-1");
+        break;
+      case "menu-item-2":
+        showContent("menu-item-2");
+        break;
+      case "menu-item-3":
+        showContent("menu-item-3");
+        break;
+      case "menu-item-4":
+        showContent("menu-item-4");
+        break;
+      case "menu-item-5":
+        showContent("menu-item-5");
+        break;
+    }
+  }
+});
+
+function showContent(nameClass) {
+  const element = document.querySelector("#content");
+  while (element.firstChild) {
+    element.removeChild(element.firstChild);
+  }
+  const newDiv = document.createElement("div");
+  element.className = "content content-style";
```

Comment:
Use `classList.add` or `classList.toggle`. It makes your code more predictable [Link](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList).
You add className for element every time when clicks nav-block. Try to rewrite this logic.

Reply:
Perhaps I misunderstood what you wrote.But after trying to use your advice, the line "div.classList.add("content content-style");" did not work in my code. Could you describe in more detail how to use "classList.add" using my code example. Thank.

Reply:
You can add that style you need `classList.add('content-style')`.Style in your `div` will contain both of your classes - old and new one

Reply:
```
// use the classList API to remove and add classes
div.classList.remove("foo");
div.classList.add("anotherclass");

// add or remove multiple classes
div.classList.add("foo", "bar", "baz");
div.classList.remove("foo", "bar", "baz");

// add or remove multiple classes using spread syntax
const cls = ["foo", "bar"];
div.classList.add(...cls); 
div.classList.remove(...cls);
```

[More examples](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList)